src/aboutwindow.cpp

```diff
@@ -12,24 +12,30 @@ AboutWindow::AboutWindow(QWidget *parent) : QDialog(parent), m_ui(new Ui::AboutW
     setWindowTitle(tr("About") + " " + qApp->applicationName());
 
     m_ui->aboutText->setText(
-            "<p>Notes was founded by <a href='https://rubymamistvalove.com'>Ruby Mamistvalove</a>, "
-            "to create an elegant yet powerful cross-platform and open-source note-taking "
-            "app.</p><a href='https://github.com/nuttyartist/notes'>Source code on "
-            "Github</a>.<br/><br/><strong>Acknowledgments</strong><br/>This project couldn't have "
-            "come this far without the help of these amazing "
-            "people:<br/><br/><strong>Programmers:</strong><br/>Alex Spataru<br/>Ali Diouri"
-            "<br/>David Planella<br/>Diep Ngoc<br/>Guilherme "
-            "Silva<br/>Thorbjørn Lindeijer<br/>Tuur Vanhoutte<br/>Waqar "
-            "Ahmed<br/><br/><strong>Designers:</strong><br/>Kevin Doyle<br/><br/>And to the "
-            "many of our beloved users who keep sending us feedback, you are an essential force in "
-            "helping us improve, thank you!<br/><br/><strong>Notes makes use of the following "
-            "third-party libraries:</strong><br/><br/>QMarkdownTextEdit<br/>"
+            "<strong>Version:</strong> " + qApp->applicationVersion()
```

Comment:
I would suggest to write `QCoreApplication::applicationVersion()` (also for the name above), since these functions are static so I think it's a bit confusing to call them like member functions.